### PR TITLE
Sleeping carp gang no longer has special equipment

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: The sleeping carp gang no longer has special equipment and an inability to use pneumatic cannons.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/pneumaticCannon.dm [hunk 1]
```diff
@@ -88,9 +88,6 @@
 	if(!istype(user) && !target)
 		return
 	var/discharge = 0
-	if(is_in_gang(user, "Sleeping Carp"))
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: This was literally only for pneumatic cannons????? I don't know why, exactly.

**Comment by KazeEspada** [comment]
Theres one in guns too.

**Comment by ChangelingRain** [comment]
That's for the martial art, which I'm leaving in.

**Comment by Robustin** [comment]
Not sure I see the advantage of this approach. All it does it let one name/graffiti stay active but at the expense of confusion over whether the Sleeping Carp is a snowflake gang or not. 

Plus if I keep the Scroll as a traitor item "Sleeping Carp Scroll" it would just reinforce that Sleeping Carp are supposed to be martial artists. Best compromise would be me renaming the scroll into something unrelated but that still leaves the confusion issue.

**Comment by kevinz000** [comment]
Sleeping carp martial art is actually interesting. If robustin's PR doesn't go through (and it's better imo) , maybe leave it in as ADMIN-ONLY so admins can still run stuff with it
And still, Robustin's PR is best PR :^)

File: code/game/objects/items/weapons/pneumaticCannon.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: There's more than this, I'm pretty sure; there's also some stuff under the projectiles module.

Comment (neersighted) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: aaaaa

**Comment by Fox-McCloud** [comment]
You may want to just disable the pneumatic cannon if you have the martial arts datum, just like with the guns....because it is a range "gun" like weapon that is pretty powerful.

File: code/game/objects/items/weapons/pneumaticCannon.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: xhuis coded pneumatic cannons and sleeping carp, and i guess cannons aren't real guns, you know the rest

**Pull request merged**